README.md

```diff
@@ -95,31 +97,33 @@ In some cases (such as recent Ubuntu systems), it may be necessary to overcome f
 
 ## Usage
 
-As mentioned above, VulcanizeDB's processes can be split into three categories: syncing, transforming and exposing data.
+VulcanizeDB's processes can be split into two categories: extracting and transforming data.
 
-### Data syncing
+### Extracting
 
-To provide data for transformations, raw Ethereum data must first be synced into VulcanizeDB. This is accomplished through the use of the `headerSync` command. This command is described in detail [here](documentation/data-syncing.md).
+Several commands extract raw Ethereum data to Postgres:
+- `headerSync` populates block headers into the `public.headers` table - more detail [here](documentation/data-syncing.md).
+- `execute` and `composeAndExecute` add configured event logs into the `public.event_logs` table.
+- `extractDiffs` pulls state diffs into the `public.storage_diff` table.
 
-### Data transformation
-Data transformation uses the raw data that has been synced into Postgres to filter out and apply transformations to specific data of interest. Since there are different types of data that may be useful for observing smart contracts, it follows that there are different ways to transform this data. We've started by categorizing this into Generic and Custom transformers:
+### Transforming
+Data transformation uses the raw data that has been synced into Postgres to filter out and apply transformations to specific data of interest.
+Since there are different types of data that may be useful for observing smart contracts, it follows that there are different ways to transform this data.
+We've started by categorizing this into Generic and Custom transformers:
 
-- Generic Contract Transformer: Generic contract transformation can be done using a built-in command, `contractWatcher`, which transforms contract events provided the contract's ABI is available. It also provides some state variable coverage by automating polling of public methods, with some restrictions. `contractWatcher` is described further [here](documentation/generic-transformer.md).
-
-- Custom Transformers: In many cases custom transformers will need to be written to provide more comprehensive coverage of contract data. In this case we have provided the `compose`, `execute`, and `composeAndExecute` commands for running custom transformers from external repositories. Documentation on how to write, build and run custom transformers as Go plugins can be found [here](documentation/custom-transformers.md).
-
-### Exposing the data
```

Comment:
Did you mean to remove this part? It's okay if you did, I'm just surprised.

Reply:
I did mean to remove the part about exposing the data, because at this point that's a responsibility that lives entirely outside of this repo - e.g. a user could choose to use postgraphile, query the db directly, or any other number of options.

The stuff about generic vs custom transformers looks like it was removed but actually just got moved down in the diff.